config/foreman.hiera/tuning/sizes/extra-extra-large.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apache::mod::passenger::passenger_max_pool_size: 240
+apache::mod::passenger::passenger_max_request_queue_size: 1000
+# TODO this is 1/10th of the base
+apache::mod::passenger::passenger_max_requests: 1000
+
+apache::mod::prefork::serverlimit: 1024
+apache::mod::prefork::maxclients: 1024
+apache::mod::prefork::maxrequestsperchild: 4000
+
+candlepin::java_opts: "-Xms1024m -Xmx8192m"
+
+postgresql::server::config_entries:
+  max_connections: 1000
+  shared_buffers: 32GB
+  work_mem: 8MB
+  checkpoint_segments: 32
```

Comment:
This will be problematic on PostgreSQL 9.5 and newer. That has dropped the setting and will prevent the server from starting up. Since only EL7 ships such an ancient version, we didn't include this setting in base. Adding this here will prevent users from using any SCL.

Reply:
When we do a Postgres upgrade, we will need to ensure we drop this from the tuning then?

Reply:
Correct. We are also knowingly breaking user setups who have chosen to use SCLs, even if it's not officially supported.

@sthirugn does this really have a large performance impact?

Reply:
Looks like https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server does strongly suggest this is a good thing.

Reply:
yes, this setting is recommended